Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/deferred/impl/AdapterResponseHelper.java

```diff
@@ -36,4 +45,76 @@ public static AdhocQueryResponse createSuccessResponse() {
         return response;
     }
 
+    public static RegistryResponseType createRegistryResponseTypeWithXdsQuerySuccess() {
+        return createRegistryResponseTypeWithXdsQueryFailure();
+    }
+
+    public static RegistryResponseType createRegistryResponseTypeWithXdsQueryFailure(String... messsages) {
```

Comment:
Why does this method exist? Its no different than "createFailureWithMessage". If you dont want a Failure, then use a Success method instead instead of calling this method with no arguments. Especially since this seems to be a copy paste of my other method with the inclusion of line 54 defaulting to success